resources/gui/layouts/shared/SettingsWindow.json

```diff
@@ -823,6 +823,172 @@
               "CheckedSound": "octave-tap-warm.wav",
               "UncheckedSound": "octave-tap-resonant.wav"
             }
+          },
+          "GraphicalUserInterfaceSettings": {
+            "Bounds": "0,104,243,57",
+            "Padding": "0,0,0,0",
+            "AlignmentEdgeDistances": "0,0,0,0",
+            "AlignmentTransform": "0,0",
+            "Margin": "0,0,0,0",
+            "RenderColor": "255,255,255,255",
+            "Alignments": "",
+            "DrawBackground": true,
+            "MinimumSize": "1,1",
+            "MaximumSize": "4096,4096",
+            "Disabled": false,
+            "Hidden": false,
+            "RestrictToParent": false,
+            "MouseInputEnabled": false,
+            "HideToolTip": false,
+            "ToolTipBackground": null,
+            "ToolTipFont": null,
+            "ToolTipTextColor": "",
+            "TextColor": "255,225,225,225",
+            "HoveredTextColor": "",
+            "ClickedTextColor": "",
+            "DisabledTextColor": "255,90,90,90",
+            "TextAlign": "Left",
+            "TextPadding": "34,0,0,0",
+            "AutoSizeToContents": false,
+            "Font": "sourcesansproblack,10",
+            "TextScale": 1.0,
+            "NormalImage": "GraphicalUserInterfaceSettings.png",
```

Comment:
Change the PNG name (also make it camelCase or snake_case `interfaceSettings.png` or `interface_settings.png`)